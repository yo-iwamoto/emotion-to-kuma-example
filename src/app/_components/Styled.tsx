import { styled } from '@kuma-ui/core';

export function Styled() {
  return <Text>Styled</Text>;
}

const Text = styled('p')`
  color: red;
  font-size: 40px;
`;
