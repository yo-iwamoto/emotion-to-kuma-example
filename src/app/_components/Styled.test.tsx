import { Styled } from './Styled';
import { render } from '@testing-library/react';

describe('Styled', () => {
  it('renders unchanged', () => {
    const { asFragment } = render(<Styled />);

    expect(asFragment()).toMatchSnapshot();
  });
});
