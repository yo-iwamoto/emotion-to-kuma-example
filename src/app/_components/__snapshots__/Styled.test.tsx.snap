// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Styled > renders unchanged 1`] = `
<DocumentFragment>
  .🐻-1582507431 {
  color: red;
  font-size: 40px;
}

  <p
    class="🐻-1582507431"
  >
    Styled
  </p>
</DocumentFragment>
`;
